soroban-env-host/benches/common/cost_types/im_map_ops.rs

```diff
@@ -1,7 +1,7 @@
 use crate::common::{util, HostCostMeasurement};
 use rand::{rngs::StdRng, seq::SliceRandom};
 use soroban_env_host::{
-    cost_runner::{ImMapImmutEntryRun, ImMapImmutEntrySample, ImMapNewRun},
+    cost_runner::{ImMapEntryRun, ImMapEntrySample, ImMapNewRun},
```

Comment:
You don't have to fix this now, but it's worth fixing eventually and if you're touching all these things: the `Im` prefix on `ImMap...` and `ImVector...` seems like it's no longer relevant. Unless you're intending to measure different costs for mutable vector and mutable vector maps in the future?

Reply:
Fixed it.